### ISSUE TITLE
blurry text rendering when adding > 1 text object to displayobject container

### ISSUE THREAD

**Issue by mreinstein** [GoodBoyDigital/pixi.js#1528]
the repro code in my display object container:

```coffeescript
t = new pixi.Text 'shutdown pump 0x4df'.toUpperCase()
t.x = 10
t.y = 10
@addChild t

@mal = new pixi.Text 'malfunction'.toUpperCase()
@mal.x = 10
@mal.y = t.y + t.height + 4
@addChild @mal
```

the result:
![screen shot 2015-03-12 at 5 09 37 pm](https://cloud.githubusercontent.com/assets/718067/6630899/81db7174-c8db-11e4-8b1f-7206c9866d04.png)

I'm running Chrome 41, and pixi 2.2.7. Is this a known issue? Am I doing something wrong?

**Comment by englercj** [comment]
Any specific renderer being used?

**Comment by mreinstein** [comment]
Sorry Chad, derp should have specified. It's using the webgl renderer.

**Comment by englercj** [comment]
@mreinstein No problem :P I'll add it to my list of things to look at this weekend. Thanks!

**Comment by alvinsight** [comment]
This might be related to #1520 ?

**Comment by mreinstein** [comment]
@englercj no the container is positioned at 190, 140

@alvinsight it's odd that in my screen shot, the 2nd text object renders crisply, but the 1st is blurry.

When I add more text objects, it seems like the last one is always crisp and the preceding ones are blurry.
![screen shot 2015-03-13 at 8 14 51 am](https://cloud.githubusercontent.com/assets/718067/6641114/29e4b754-c959-11e4-8156-dc4e66e7a2f6.png)

**Comment by mreinstein** [comment]
@englercj this seems to be fixed in `2.2.8` !
